src/Frontend/Page/Topic.hs

```diff
@@ -162,21 +162,26 @@ instance FormPage CreateTopic where
         semanticDiv p $ do
             div_ [class_ "container-main popup-page"] $ do
                 div_ [class_ "container-narrow"] $ do
-                    h1_ [class_ "main-heading"] "Create Topic"
-                    DF.form v fa $ do
-                        label_ $ do
-                            span_ [class_ "label-text"] "Wie soll der Titel des Themas lauten?"
-                            inputText_ [class_ "m-small", placeholder_ "z.B. Computerraum"]
-                                "title" v
-                        label_ $ do
-                            span_ [class_ "label-text"] "Beschreiben Sie das Thema"
-                        -- FIXME I want a placeholder here too
-                            DF.inputTextArea Nothing Nothing "desc" v
-                        label_ $ do
-                            span_ [class_ "label-text"] "Fügen Sie weitere wilde dem neuen Thema hinzu"
-                            formPageIdeaSelection v ideas
-                        footer_ [class_ "form-footer"] $ do
-                            DF.inputSubmit "Veröffentlichen"
+                    h1_ [class_ "main-heading"] "Thema erstellen"
+                    DF.form v fa $ createOrEditTopic v ideas
+
+createOrEditTopic :: Monad m => View (HtmlT m ()) -> [Idea] -> HtmlT m ()
+createOrEditTopic v ideas = do
+    label_ $ do
+        span_ [class_ "label-text"] "Wie soll der Titel des Themas lauten?"
+        inputText_ [class_ "m-small", placeholder_ "z.B. Computerraum"]
+            "title" v
+    label_ $ do
+        span_ [class_ "label-text"] "Beschreiben Sie das Thema"
+    -- FIXME I want a placeholder here too
+        DF.inputTextArea Nothing Nothing "desc" v
```

Comment:
> On Edit topic the exiting text of the topic should be rendered as the content in the  `inputTextArea`.

yes, but in the 'create' mode, it doesn't.  and we don't want to have it as the default value either.  there is a 'placeholder_' attribute, but it's not supported by DF.inputTextArea, so we need to write another wrapper.